### PR TITLE
fix(condo): DOMA-10006 add throttling

### PR DESCRIPTION
**Pull request by toplenboren**

### CODE REVIEW

**Review by pahaz — APPROVED** [review]

File: apps/condo/domains/news/components/RecipientCounter.tsx [hunk 1]
```diff
@@ -237,11 +238,13 @@ export const RecipientCounter: React.FC<RecipientCounterProps> = ({ newsItemScop
         },
     )
 
+    const throttledGetCounters = useCallback(throttle(getCounters, 1500), [throttle])
```
Comment (SavelevMatthew) [inline apps/condo/domains/news/components/RecipientCounter.tsx:241]: Why `getCounters` not in deps array? To avoid callback rerender?

Comment (SavelevMatthew) [reply apps/condo/domains/news/components/RecipientCounter.tsx:241]: btw, throttle is imported function, its not necessary to put it in reps

File: apps/condo/domains/news/components/RecipientCounter.tsx [hunk 2]
```diff
@@ -266,6 +269,9 @@ export const RecipientCounter: React.FC<RecipientCounterProps> = ({ newsItemScop
 
     const isLoadingCounters = isCountersLoading || !counters
 
+    console.log('Recipient counter', { isLoadingCounters, isCountersLoading, counters })
```
Comment (SavelevMatthew) [inline apps/condo/domains/news/components/RecipientCounter.tsx:272]: ?

File: apps/condo/domains/news/components/RecipientCounter.tsx (hunk 1 above)
Comment (SavelevMatthew) [reply apps/condo/domains/news/components/RecipientCounter.tsx:241]: And why 1500? I'm almost sure we can reproduce error consistently with this value

**Pull request merged**